web-ui/src/components/guild-results/GuildSummaryCard.jsx

```diff
@@ -143,7 +143,6 @@ const GuildSummaryCard = ({ guild, index, isOpen, onGuildSelect }) => {
         style={{
           color: guild.community ? 'var(--oci-orange)' : 'var(--oci-light-blue)'
         }}
-        title={`This is a ${guild.community ? 'Community' : 'Guild'}.`}
```

Comment:
Why do you want to remove this?

Reply:
There's an AC in the story that triggered adding this. It asks for the icons to have a tooltip.

Reply:
Michael requested we remove this for now to unblock CI.